### PR TITLE
groups: add option to configure font name

### PR DESCRIPTION
**Pull request by dlealmt**
Hey, this is a very small PR that only adds the ability to set the group bar's font name. I have no idea about what I'm doing, since I don't know C++, but this is working fine for me, including if I change the font name on the fly, and even if I use the name of a non-existent font.

I'll be happy to make any changes you deem necessary, within my very limited ability.

### CODE REVIEW

File: src/config/ConfigManager.cpp [hunk 1]
```diff
@@ -129,6 +129,7 @@ void CConfigManager::setDefaultVars() {
     configValues["group:insert_after_current"].intValue = 1;
     configValues["group:focus_removed_window"].intValue = 1;
 
+    configValues["group:groupbar:font_name"].strValue          = "Sans";
```
Comment (vaxerski) [inline src/config/ConfigManager.cpp:132]: font_family

**Review by vaxerski — COMMENTED** [review]
also you need to slap a `clang-format` on this.

And a wiki mr needed for the new opt

File: src/config/ConfigManager.cpp (hunk 1 above)
Comment (dlealmt) [reply src/config/ConfigManager.cpp]: ✅

File: src/render/decorations/CHyprGroupBarDecoration.cpp [hunk 2]
```diff
@@ -179,6 +179,7 @@ CTitleTex::CTitleTex(CWindow* pWindow, const Vector2D& bufferSize) {
     const auto         CAIROSURFACE = cairo_image_surface_create(CAIRO_FORMAT_ARGB32, bufferSize.x, bufferSize.y);
     const auto         CAIRO        = cairo_create(CAIROSURFACE);
 
+    static auto const PTITLEFONTFAMILY = &g_pConfigManager->getConfigValuePtr("group:groupbar:font_family")->strValue;
```
Comment (vaxerski) [inline src/render/decorations/CHyprGroupBarDecoration.cpp:182]: misses a `*` and a clang-format

Comment (dlealmt) [reply src/render/decorations/CHyprGroupBarDecoration.cpp]: Apologies, I missed this comment. But I think this is done now.

**Review by vaxerski — APPROVED** [review]
lgtm, please make a wiki mr, then I'll merge

**Comment by dlealmt** [comment]
> lgtm, please make a wiki mr, then I'll merge

Done.

**Comment by vaxerski** [comment]
thanks!

**Pull request merged**